includes/tracks/events/class-wc-importer-tracking.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * WooCommerce Import Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class adds actions to track usage of WooCommerce Imports.
+ */
+class WC_Importer_Tracking {
+	/**
+	 * Init tracking.
+	 */
+	public static function init() {
+		add_action( 'product_page_product_importer', array( __CLASS__, 'track_product_importer' ) );
+	}
+
+	/**
+	 * Route product importer action to the right callback.
+	 *
+	 * @return void
+	 */
+	public static function track_product_importer() {
+		if ( ! isset( $_REQUEST['step'] ) ) {
```

Comment:
The linter is complaining here below, `WARNING | [ ] Processing form data without nonce verification.`

Reply:
Fixed in 767f435, using https://github.com/woocommerce/woocommerce/blob/e9f04908293048cf3a154a0771f5db61b01258e2/includes/admin/importers/class-wc-product-csv-importer-controller.php#L262 as a guide.